pkg/kubecost/asset.go

```diff
@@ -1772,6 +1772,7 @@ type Node struct {
 	RAMCost      float64
 	Discount     float64
 	Preemptible  float64
+	Pool         string
```

Comment:
This needs a version annotation, and you need to update the assets version for bingen - anytime a primary pipeline data structure is changed, you have to do this. Read docs here: 

https://github.com/opencost/opencost/blob/develop/pkg/kubecost/bingen.go

Reply:
Also, better docs here: github.com/kubecost/bingen